students/morgan/session03/list_lab.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+# $ ./list_lab.py
+# chmod +x list_lab.py
+
+fruit = ['Apples', 'Pears', 'Oranges', 'Peaches']
+
+print(fruit)
+
+new_fruit = input("What's fruit would you like to add?\n"
+            "> ")
+
+fruit.append(new_fruit)
+
+print(fruit)
+    
+number = int(input("What's your number?\n"
+            "> "))
+
+print(number, fruit[number-1])
+
+newest_fruit = []
+newest_fruit.append(input("What's fruit would you like to add?\n"
+            "> "))
+print(newest_fruit)
+added_fruit_list = newest_fruit + fruit
+print(added_fruit_list)
+
+added_fruit_list.insert(0, input("What's fruit would you like to add?\n"
+            "> "))
+
+for x in range(len(added_fruit_list)):
+    if str.lower(added_fruit_list[x][0:1]) == 'p':
+        print(added_fruit_list[x])
+
+
+# Series 2
+
+print(added_fruit_list)
+
+added_fruit_list = added_fruit_list[:-1]
+print(added_fruit_list)
+
+
+remove_fruit = input("What's fruit would you like to remove?\n"
+            "> ")
+
+print(remove_fruit)
+print(added_fruit_list[0])
+
+for thing in added_fruit_list:
+    if str.lower(thing) == str.lower(remove_fruit):
+        added_fruit_list.remove(thing)
+
```

Comment:
Be very careful deleting objects from a list that you are iterating through. Mostly, don't do it. An interesting alternative Chris came up with:
```
while ans in d_fruits:
    d_fruits.remove(ans)
```
This doesn't address the lowercase issue, however. Creating a copy of the list, minus the fruit to be removed is one option:
```
new_list = [fruit for fruit in added_fruit_list if str.lower(fruit) != str.lower(remove_fruit)]
```